compiler/src/dmd/cli.d

```diff
@@ -910,45 +911,64 @@ dmd -cov -unittest myprog.d
 
     /// Returns all available reverts
     static immutable reverts = [
-        Feature("dip25", "useDIP25", "revert DIP25 changes https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md", true, true),
+        Feature("dip25", "useDIP25", "revert DIP25 changes",
+        "https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md", true, true),
         Feature("dip1000", "useDIP1000",
-                "revert DIP1000 changes https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1000.md (Scoped Pointers)"),
+                "revert DIP1000 changes (Scoped Pointers)",
+                "https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1000.md"),
         Feature("intpromote", "fix16997", "revert integral promotions for unary + - ~ operators"),
         Feature("dtorfields", "dtorFields", "don't destruct fields of partially constructed objects"),
     ];
 
     /// Returns all available previews
     static immutable previews = [
+        // Note: generally changelog entries should be added to the spec
         Feature("dip25", "useDIP25",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md (Sealed references)", true, true),
+            "implement Sealed References DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md", true, true),
         Feature("dip1000", "useDIP1000",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1000.md (Scoped Pointers)"),
+            "implement Scoped Pointers DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1000.md"),
         Feature("dip1008", "ehnogc",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1008.md (@nogc Throwable)"),
+            "implement @nogc Throwable DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1008.md"),
         Feature("dip1021", "useDIP1021",
-            "implement https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1021.md (Mutable function arguments)"),
-        Feature("bitfields", "bitfields", "add bitfields https://github.com/dlang/dlang.org/pull/3190"),
-        Feature("fieldwise", "fieldwise", "use fieldwise comparisons for struct equality"),
+            "implement Mutable Function Arguments DIP",
+            "https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1021.md"),
+        Feature("bitfields", "bitfields", "add C-like bitfields",
+            "https://github.com/dlang/dlang.org/pull/3190"),
+        Feature("fieldwise", "fieldwise", "use fieldwise comparisons for struct equality",
+            "https://dlang.org/changelog/2.085.0.html#no-cmpsb"),
         Feature("fixAliasThis", "fixAliasThis",
-            "when a symbol is resolved, check alias this scope before going to upper scopes"),
+            "when a symbol is resolved, check alias this scope before going to upper scopes",
+            "https://github.com/dlang/dmd/pull/8885"),
```

Comment:
There didn't seem to be a better link to use than this PR. At least it mentions the 2 bugzilla issues it fixes.